### PR TITLE
Add the XI support for iOS, remove Bugsnag from the soltuion

### PR DESCRIPTION
**Pull request by masterrr**
Tested in Release/Debug for Ross/Joey. Users/Crashes/Warnings etc. reporting :+1: 
Closing https://github.com/toggl/mobile/issues/767

### CODE REVIEW

File: Phoebe/Logging/BaseLogger.cs [hunk 1]
```diff
@@ -29,7 +28,7 @@ private void Process (LogLevel level, string tag, string message, Exception exc
         {
             LogToConsole (level, tag, message, exc);
             LogToFile (level, tag, message, exc);
-            LogToBugsnag (level, tag, message, exc);
+            LogToXI (level, tag, message, exc);
```
Comment (monday8am) [inline Phoebe/Logging/BaseLogger.cs]: @masterrr Change the name of this method to a more generic name: LogToLoggerClient or LogToRemoteBugTracker

**Comment by monday8am** [comment]
:+1: ok for me. 

Just the change of LogToXI name and to update the XI version to 1.10.4 and delete Bugsnag libraries.

**Comment by masterrr** [comment]
@monday8am changed the method name, removed Bugsnag packages

**Comment by monday8am** [comment]
:+1:

**Pull request merged**